assets/src/libs/Helper.js

```diff
@@ -0,0 +1,47 @@
+/* -------------------------------------------------------------------------- */
+
+/**
+* Split ticket requester identities to multiple wholes
+*
+* @param {Object} identities
+*
+* @return {Object}
+*
+*/
+export function splitIdentities(identities) {
+  let emails = [];
+  let phones = [];
+  let primaries = {
+    email: '',
+    phone: '',
+  };
+
+  identities.forEach((identity) => {
+    switch (identity.type) {
+      case 'email': {
+        if (identity.primary) {
+          primaries.email = identity.value;
+        }
+
+        emails.push(identity.value);
+      } break;
+      case 'phone_number': {
+        if (identity.primary) {
+          primaries.phone = identity.value;
+        }
+
+        phones.push(identity.value);
+      } break;
+    }
+  });
+
+  return {
+    all: {
+      emails: emails,
```

Comment:
es6 doesn't need that. the value has the same name as the property so you can just use `emails` without `: emails` (same reason why I'm using it for `template`)